### ISSUE TITLE
[White Label - Videocard]: Keyboard focus is not visible all four sides on video controls present under video.

### ISSUE THREAD

**Issue by Kusum199** [microsoft/BotFramework-WebChat#5705]
### Is it an issue related to Adaptive Cards?

No.

### What is the PWD impact?

When the keyboard focus indicator is not clearly visible on all four sides of the video controls, keyboard users may struggle to identify which control is currently active, leading to confusion, navigation errors, and difficulty operating the video player.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Keyboard focus indicator

### How to reproduce the issue?

1. Open URL: https://compulim.github.io/webchat-loader/ and sign in using valid credentials.
2. Navigate to web chat page and select 4.18.1-main.20240927.ea7a875 (30/9/2024) version and then select mockBot link under present label and then select open web chat in new window link at bottom.
3.Navigate to white label experience chat page and type Videocard word in type your message text box.
4.Navigate to the search results.
5.Verify that Keyboard focus is not visible all four sides on video controls present under video.

### What do you expect?

The keyboard focus indicator should be clearly visible on all four sides with dark colour contrast of the video controls under the video, ensuring users can easily identify the currently focused element during keyboard navigation.

### What actually happened?

Keyboard focus is not visible all four sides on video controls present under video.

### Do you have any screenshots or recordings to repro the issue?

https://github.com/user-attachments/assets/18d96709-5581-48fc-97a1-8e6842d3ac3b

### Adaptive Card JSON

```json

```
